tests/Browser/ContactTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Je mist tests, je dekt zo niet alle acceptatiecriteria. Je test alleen het happy path. Maak ook nog tests om te testen wat er gebeurt als het geen geldig email is of het bericht langer is dan 500 karakters. En een test om te checken of het bericht werkelijk op de website wordt opgeslagen.